wapitiCore/definitions/xss.py

```diff
@@ -25,6 +25,8 @@
 
 SHORT_NAME = _("XSS")
 
+WSTG_CODE = ["WSTG-INPV-02"]
```

Comment:
we should keep stored_xxs.py ?

https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/07-Input_Validation_Testing/02-Testing_for_Stored_Cross_Site_Scripting

Reply:
Uh, do you want to rename the file ?

Reply:
I tried to rename the xss to stored_xss, but the program doesn't work anymore, it seems that the attack modules are linked to the definitions by their id.